public/index.html

```diff
@@ -16,11 +18,145 @@
     <!-- initialize the SDK after all desired features are loaded -->
     <script defer src="/__/firebase/init.js"></script>
 
-
 </head>
 
 <body>
-    Hello 12
+    
+    <nav class="navbar navbar-expand-lg sticky-top navbar-dark bg-dark">
+        
+            <a class="navbar-brand text-white px-2" style="font-weight: bold;" href="index.html">
+                <img src="image/logo.png" width="30" height="30" alt="" loading="lazy">
+                Work Trolly
+            </a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText"
+                aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon text-white"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarText">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active pr-5 py-1">
+                        <button type="button" class="btn btn-outline-info" id="createNewTaskButton">Create New
+                            Task</button>
+                    </li>
+                    <li class="nav-item pr-5 py-1">
+                        <button type="button" class="btn btn-outline-info">All Task</button>
+                    </li>
+                    <li class="nav-item py-1">
+                        <button class="btn btn-primary" type="button">Login</button>
+                    </li>
+                </ul>
+
+            </div>
+        
+    </nav>
+
+    <!-- create new task   ------------------------------------------------------->
+
+    <div class="row my-5 mx-5" id="createNewTask">
+        <div class="container bg-light py-3 px-5">
```

Comment:
Containers are always used outside the row.

Reply:
I mean top layer is container always

Reply:
okay, I will make the top layer as a container.